reup/revive/templates/revive/index.html

```diff
@@ -7,12 +7,15 @@ <h1 class="display-4">reup</h1>
         <p class="lead">Simple tool to search for documents by the old ID.</p>
         <hr class="my-4">
 
-        <p>Please fill in the ID</p>
+        <p>
+            Please fill in the ID or Url <br/>
+            <small>If an url is provided the first digits group will be used</small>
```

Comment:
Let's give some examples instead of writing help text: one example with just the numeric id, another with the full URL.

Reply:
Updated.